HCSP.js

```diff
@@ -0,0 +1,228 @@
+{
+    "translatorID": "3e375db7-5f6e-4861-ab8b-3ddb0f4accc9",
+    "label": "HCSP",
+    "creator": "Joris Muller",
+    "target": "http://www.hcsp.fr",
+    "minVersion": "2.1",
+    "maxVersion": "",
+    "priority": 100,
+    "inRepository": true,
+    "translatorType": 4,
+    "browserSupport": "g",
+    "lastUpdated": "2015-08-04 14:29:59"
+}
+
+/*
+HCSP Translator
+A Zotero's Translator for the website of "Haut Conseil de Santé Publique"
+Copyright (C) 2015 Joris Muller <jomuller@jom.link>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+// Usage of Zotero.Utilities.xpath() function. Exist only since Zotero 2.1
+// https://www.zotero.org/support/dev/client_coding/changes_in_zotero_2.1#xpath_utility_functions
+// Then minimal version of Zotero needed is 2.1
+
+// TODO: Use camelStyle for variables to be consistent with Zotero coding style
+
+function detectWeb(doc, url) {
+    /* The URLs have upper or lower case on the website
+    depending the way you access to a page */
+    if (url.match("[Ee]xplore.cgi/avisrapportsdomaine")) {
```

Comment:
That is actually a RegExp (strings are converted to RegExp in `match`), but explore.cgi part, and even both "avisrapports..." parts should simply be moved to the target RegExp above.

Also, iirc, we do case insensitive matching with the target RegExp